sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/authoring/aio/_operations/_operations.py

```diff
@@ -63,7 +63,9 @@ def list_projects(
     ) -> AsyncIterable[JSON]:
         """Gets all projects for a user.
 
-        Gets all projects for a user.
+        See
+        https://learn.microsoft.com/rest/api/cognitiveservices/questionanswering/question-answering-projects/list-projects
+        for more information.
```

Comment:
From your README.md I see you simply replace the description, but looking at these examples I can see why. I presume that typical doc comment renderings e.g., tooltips show would otherwise have been duplicate e.g.,
> **Gets all projects for a user**
>
> Gets all projects for a user
If so, this is a definite improvement.

Reply:
I didn't notice the duplicate docstrings until this PR. It seems if `summary` is used in the swagger, then we duplicate the text in the docstring. If `description` is used, then it only shows up once. @iscai-msft - do you know if this intentional by the python generator?

Reply:
Opened an issue in autorest: https://github.com/Azure/autorest.python/issues/1482